src/node/Main.hs

```diff
@@ -4,18 +4,20 @@
 {-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeOperators       #-}
+{-# LANGUAGE TemplateHaskell     #-}
 
 module Main
        ( main
        ) where
 
+import           Development.GitRev  (gitBranch, gitHash)
```

Comment:
According to the [style-guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md) `Universum` should be a separate block of imports and 3rd-party libraries should be all in the same block.